user_docs/en/userGuide.t2t

```diff
@@ -34,6 +34,15 @@ Major highlights include:
 - The ability to highlight the system focus
 -
 
++ System Requirements +[SystemRequirements]

```

Comment:
This used to be a top level header, which means every subsequent item below this used to be under "1. Introduction" now are under "2. System Requirements". I suggest system requirements become a subsection under introduction

```suggestion
++ System Requirements ++[SystemRequirements]
```